website/docs/d/schema_site_service_graph.html.markdown

```diff
@@ -34,3 +34,8 @@ data "mso_schema_site_service_graph" "example" {
 
 * `service_node` - (Read-Only) List of maps to provide Site level Node association.
     * `device_dn` - (Read-Only) Dn of device associated with the service node of the Service Graph.
+    * `provider_connector_type` - (Read-Only) Connector type of the provider type of the service node. This parameter is only applicable for cloud sites. This parameter is only applicable for third_party_load_balancer and third-party firewall service nodes, when the template is attached to cloud sites.
```

Comment:
when firewall is separate input should this be mentioned in provider_connector_type?